pkgs/servers/web-apps/vikunja/frontend.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, lib, fetchFromGitea, mkYarnPackage, yarn, apiBase ? "/api/v1", ... }:
+
+let
+  version = "0.17.0";
+
+  src = fetchFromGitea {
+    domain = "kolaente.dev";
+    owner = "vikunja";
+    repo = "frontend";
+    rev = "v${version}";
+    sha256 = "sha256-/KCpsPEhCJz+1pLisSviFP/VCiXUD6N9l1ZragOd2Go=";
+  };
+  frontend-modules = mkYarnPackage rec {
+    name = "vikunja-frontend-modules";
+    inherit version src;
+
+    packageJSON = ./frontend-package.json;
```

Comment:
Is it really required to keep the `package.json` and generated `yarn2nix` files in the repo? AFAIU `yarn2nix` will regenerate them based on the content of the repo itself so these are rather redundant.

Reply:
No longer important, since reverted to pre-build version.